client/public/index.html

```diff
@@ -1,14 +1,16 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <!-- Required meta tags -->
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-    
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+
```

Comment:
It looks like all the changes in this file are just caused by reformatting (there are no actual changes to the code). Try to avoid committing changes like this as part of other work since it makes it harder to work out what is actually making a difference to the app behaviour.